src/Jackett.Common/Models/IndexerConfig/Bespoke/ConfigurationDataMyAnonamouse.cs

```diff
@@ -7,16 +8,30 @@ internal class ConfigurationDataMyAnonamouse : ConfigurationData
     {
         public StringConfigurationItem MamId { get; private set; }
         public DisplayInfoConfigurationItem MamIdHint { get; private set; }
-        public BoolConfigurationItem ExcludeVip { get; private set; }
-        public DisplayInfoConfigurationItem Instructions { get; private set; }
+        public SingleSelectConfigurationItem SearchType { get; private set; }
+        public BoolConfigurationItem SearchInDescription { get; private set; }
+        public BoolConfigurationItem SearchInSeries { get; private set; }
+        public BoolConfigurationItem SearchInFilenames { get; private set; }
 
         public ConfigurationDataMyAnonamouse()
         {
             MamId = new StringConfigurationItem("mam_id");
             MamIdHint = new DisplayInfoConfigurationItem("mam_id instructions", "Go to your <a href=\"https://www.myanonamouse.net/preferences/index.php?view=security\" target=\"_blank\">security preferences</a> and create a new session for the IP used by the Jackett server. Then paste the resulting mam_id value into the mam_id field here.");
-            ExcludeVip = new BoolConfigurationItem("Exclude VIP torrents");
-            Instructions = new DisplayInfoConfigurationItem("", "For best results, change the 'Torrents per page' setting to 100 in your Profile => Torrent tab.");
+            SearchType = new SingleSelectConfigurationItem(
+                "Search Type",
+                new Dictionary<string, string>
+                {
+                    { "all", "All torrents" },
+                    { "active", "Only active" },
+                    { "fl", "Freeleech" },
+                    { "fl-VIP", "Freeleech or VIP" },
+                    { "VIP", "VIP torrents" },
+                    { "nVIP", "Torrents not VIP" },
+                })
+            { Value = "all" };
+            SearchInDescription = new BoolConfigurationItem("Search text in the description") { Value = false };
+            SearchInSeries = new BoolConfigurationItem("Search text in the series") { Value = false };
+            SearchInFilenames = new BoolConfigurationItem("Search text in the filenames") { Value = false };
```

Comment:
Would adding 'Also' to the start of these make it clearer that they're in addition, not instead of (e.g. `Also search text in the description`)?